### ISSUE TITLE
Fix filtering for string id's 

### ISSUE THREAD

**Issue by RichardLindhout** [web-ridge/gqlgen-sqlboiler#14]
We need to fix filters when model has string primary id's instead of uints
https://github.com/web-ridge/gqlgen-sqlboiler/issues/12#issuecomment-625532393

**Comment by troian** [comment]
Yah, I think I've just got this one
```
graph/helpers/convert_input.go:186:3: cannot use boilergql.IDToNullBoiler(boilergql.PointerStringToString(m.MeetingID)) (type null.Uint) as type null.String in field value
graph/helpers/convert_input.go:254:3: cannot use boilergql.IDToNullBoiler(boilergql.PointerStringToString(m.MeetingID)) (type null.Uint) as type null.String in field value
```

**Comment by RichardLindhout** [comment]
That's another one I think since this is about the filter.go which does not complain as in compile issues but does not work.

**Comment by RichardLindhout** [comment]
Strange that it still generates IDToNullBoiler since you're using string id's it should not use the ID functions anymore.

**Comment by RichardLindhout** [comment]
Fixed in: v2.0.7
